src/frontend/components/UI/Sidebar/components/SidebarItem/index.css

```diff
@@ -0,0 +1,111 @@
+.Sidebar {
+  .Sidebar__item {
+    --sidebar-item-inactive-text-color: var(
+      --navbar-inactive,
+      var(--navbar-accent)
+    );
+    --sidebar-hover-text-color: var(--text-hover);
+    --sidebar-item-active-text-color: var(
+      --navbar-active,
+      var(--accent-overlay, var(--accent))
+    );
+    --sidebar-active-item-background: var(--navbar-active-background);
```

Comment:
this file is mostly extraction of `Sidebar__item` styles that were in other places, but I also added these new custom properties to help themming